packages/commerce-server/src/determine-coupon-to-apply.ts

```diff
@@ -297,6 +298,7 @@ const getBulkCouponDetails = async (params: GetBulkCouponDetailsParams) => {
     newPurchaseQuantity: quantity,
     prismaCtx,
   })
+  const consideredBulk = seatCount > 1
```

Comment:
Here we determine if the purchase will be considered bulk based on the qualified seat count